src/main/resources/com/mabl/integration/jenkins/MablStepBuilder/help-restApiKeyName.html

```diff
@@ -0,0 +1,5 @@
+<div>
+    mabl API key name for your workspace. The key is kept safely in Jenkins credentials as a Username with
```

Comment:
Same comment as above on name/password type of secret.

Reply:
Again, this is just the least common denominator between 1.x and 2.x. We could make the code behave differently for 2.x.

Reply:
Should the label now be `API Key Credential`, rather than `API Key Name` ?

Reply:
If someone uses the credential wrong, will they get a helpful error message? (e.g. secret is in the `Username`)

Reply:
Good point on using credential instead of name. I will update accordingly.